config-halo2.toml

```diff
@@ -0,0 +1,28 @@
+# config-example.toml
+
+[build]
+# For iOS device_type can be x86_64, simulator, device
+ios_device_type = "simulator" # Options: x86_64, simulator, device
+# For Android device_type can be x86_64, x86, arm, arm64
+android_device_type = "arm64" # Options: x86_64, x86, arm, arm64
+
+# debug is for Rust library to be in debug mode and release for release mode
+# We recommend release mode by default for performance
+build_mode = "release" # Options: debug, release
+
+[circuit]
+kind = "halo2"                    # Proving system options: circom (default), halo2
+dir = "mopro-core/examples/halo2/halo2-fibonacci" # Directory of the circuit
+name = "fibonacci"                # Name of the circuit
+
+[dylib]
```

Comment:
I guess these options are not needed for Halo2. Should we remove dylib and witness from this config?

Reply:
I tried to minimize changes to the original file structure. Will keep this for the improvement PR along with other more serious changes, like circom feature flags, etc.

Reply:
Created an issue for that, will try to do it some time next week